src/syscheckd/syscheck.c

```diff
@@ -78,11 +78,11 @@ void read_internal(int debug_level)
 void fim_initialize() {
     // Create store data
 #ifndef WIN32
-    fim_db_init(syscheck.database_store, syscheck.sync_interval, syscheck.file_limit, NULL,
+    fim_db_init(syscheck.database_store, syscheck.sync_interval, syscheck.file_limit, fim_send_sync_state,
                 loggingFunction);
```

Comment:
you could write this like `fim_send_sync_state, loggingFunction);` to improve the style